.travis.yml

```diff
@@ -24,8 +24,6 @@ go_import_path: sigs.k8s.io/kubebuilder
 services: docker
 
 # Install must be set to prevent default `go get` to run.
-# The dependencies have already been vendored by `dep` so
-# we don't need to fetch them.
 install: skip
```

Comment:
Do we still need `install: skip`? I think we should ask the one that wrote the comment you deleted why it was used and think if that applies for v2 tests.

Reply:
@droot That comment was added by you a couple ages ago. Should it stay for v2 now that it uses gomodules?

Reply:
It was removed since as no longe need.